pennylane/beta/tapes/qnode.py

```diff
@@ -418,6 +422,18 @@ def to_torch(self, dtype=None):
 
             self.dtype = dtype or self.dtype or TorchInterface.dtype
 
+            if self.dtype is np.complex128:
+                torch_version = list(map(int, torch.__version__.split(".")[:2]))
+                min_version_for_complex = torch_version[0] >= 1 and torch_version[1] >= 6
+
+                if min_version_for_complex:
+                    self.dtype = torch.complex128
+                else:
+                    raise QuantumFunctionError(
+                        "Version 1.6.0 or above of PyTorch must be installed"

```

Comment:
Since this is something that might apply elsewhere as well, maybe we could have this check closer to the codebase of torch interface?